### ISSUE TITLE
Pause Overrides MRR.

### ISSUE THREAD

**Issue by Yamanipanuchi** [RainbowMiner/RainbowMiner#1176]
Dont know if this is by design. But I have my systems pause for a few hours a day due to heat. I thought MRR would over ride this pause. It looks as if it does not.

Is it possible MRR can override a scheduled pause?

**Comment by RainbowMiner** [comment]
Actually, it should pick up rentals, even when in pause. Did you get a rental and it didn't start?

**Comment by Yamanipanuchi** [comment]
yes, it was a really crappy rental so i just let the pause stay in place.

**Comment by Yamanipanuchi** [comment]
Scheduled pause is still pausing MRR rentals. What is odd is that sometimes it will break the pause. I find it from time to time mining a rental during scheduled pause. But this morning it paused in the middle of a rental.

Dont know if this is something that might need to be an option? Something like "MRR Overrides Pause"?
